CrtanjeLikova/GeometrijskiLikovi/Elipsa.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Vsite.CSharp.Metode.GeometrijskiLikovi
+{
+    internal class Elipsa:GeometrijskiLik
```

Comment:
U Visual Studiju kliknite desnom tipkom na Elipsu, Quick actions and refactoring i odaberete "Implement abstract class".
Nakon toga bi se projekt trebao buildati bez pogreške.
Naravno, bilo bi zgodno i da napravite domaću zadaću :-).